api/api-api-support-v1/src/main/java/com/thoughtworks/go/apiv1/apisupport/ApiSupportControllerV1.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.apiv1.apisupport;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.thoughtworks.go.api.ApiController;
+import com.thoughtworks.go.api.ApiVersion;
+import com.thoughtworks.go.api.spring.ApiAuthenticationHelper;
+import com.thoughtworks.go.server.service.result.HttpLocalizedOperationResult;
+import com.thoughtworks.go.server.service.support.ServerStatusService;
+import com.thoughtworks.go.spark.Routes;
+import com.thoughtworks.go.spark.spring.SparkSpringController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import spark.Request;
+import spark.Response;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static spark.Spark.*;
+
+@Component
+public class ApiSupportControllerV1 extends ApiController implements SparkSpringController {
+
+    private final ApiAuthenticationHelper apiAuthenticationHelper;
+    private ServerStatusService serverStatusService;
+
+    private Gson gson = new GsonBuilder().setPrettyPrinting().serializeNulls().create();
+
+    @Autowired
+    public ApiSupportControllerV1(ApiAuthenticationHelper apiAuthenticationHelper, ServerStatusService serverStatusService) {
+        super(ApiVersion.v1);
+        this.apiAuthenticationHelper = apiAuthenticationHelper;
+        this.serverStatusService = serverStatusService;
+    }
+
+    @Override
+    public String controllerBasePath() {
+        return Routes.Support.BASE;
+    }
+
+    @Override
+    public void setupRoutes() {
+        path(controllerBasePath(), () -> {
+            before("", mimeType, this::setContentType);
+            before("/*", mimeType, this::setContentType);
+            before("", this.mimeType, this.apiAuthenticationHelper::checkAdminUserAnd403);
```

Comment:
check for `/*` too?
` before("/*", this.mimeType, this.apiAuthenticationHelper::checkAdminUserAnd403);`